packages/test/src/test-otel.ts

```diff
@@ -9,21 +9,27 @@ import * as opentelemetry from '@opentelemetry/sdk-node';
 import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
 import * as workflows from './workflows';
 import { WorkflowClient } from '@temporalio/client';
+import { OpenTelemetryWorkflowClientCallsInterceptor } from '@temporalio/interceptors-opentelemetry/lib/client';
+import {
+  makeWorkflowExporter,
+  OpenTelemetryActivityInboundInterceptor,
+} from '@temporalio/interceptors-opentelemetry/lib/worker';
+import { OpenTelemetryDependencies } from '@temporalio/interceptors-opentelemetry/lib/workflow';
 
 // Un-skip this test and run it by hand to inspect outputted traces
 test.skip('Otel spans connected', async (t) => {
```

Comment:
How much work would it be to run this and add assertions?
Doesn't have to be in this PR but I'd at least track it.